packages/task/src/node/task-server.ts

```diff
@@ -170,6 +170,16 @@ export class TaskServerImpl implements TaskServer, Disposable {
         }
     }
 
+    async getCode(id: number): Promise<number | undefined> {
```

Comment:
could we rename `id` into `taskId` ? I didn't understand what this function does until I saw the `interface TaskServer`